src/components/LrRouteComponents.vue

```diff
@@ -0,0 +1,41 @@
+<!--
+* CORTX-CSM: CORTX Management web.
+* Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as published
+* by the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Affero General Public License for more details.
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <https://www.gnu.org/licenses/>.
+* For any questions about this software or licensing,
+* please email opensource@seagate.com.
+-->
+<template>
+  <div>
+    <router-view v-if="isDashboard" />
```

Comment:
this condition needs change since the card view in not required for dashboard, login , and some more pages. also need condition for sidenav and header as they would hidden for login and other pages.

Reply:
I understand the flaw in this. We can take this as a separate task.